packages/analyticsdx-template-lint/src/schemas/readiness-schema.json

```diff
@@ -305,7 +337,8 @@
         "properties": {
           "field": {
             "type": "string",
-            "description": "The name of the field. This can use ${...} expressions."
+            "description": "The name of the field. This can use ${...} expressions.",
+            "minLength": 1
```

Comment:
You can technically put in a empty string for `field` but it won't work at runtime, so warn here.